app/components/pagination.hbs

```diff
@@ -1,16 +1,16 @@
-{{#if (gt @meta.page 1)}}
+{{#if (gt @page 1)}}
   <span class="pagination">
     <LinkTo
       @route={{@route}}
-      @query={{hash page=(dec @meta.page)}}
+      @query={{hash page=(dec @page)}}
     >
       {{t "pagination_prev"}}
     </LinkTo>
   </span>
 {{/if}}
-{{#each (range 1 (inc @meta.total_pages)) as |number|}}
-  {{#if (eq number @meta.page)}}
-    {{number}}
+{{#each (range 1 (inc @totalPages)) as |number|}}
+  {{#if (eq-number number @page)}}
```

Comment:
Unfortunately since these are not the same type (string vs int?) I had to create a new helper to coerce the types to ints.